tools/mpremote/mpremote/commands.py

```diff
@@ -20,16 +20,17 @@ def do_connect(state, args=None):
         if dev == "list":
             # List attached devices.
             for p in sorted(serial.tools.list_ports.comports()):
```

Comment:
I think is more efficient to filter on the bus loaction , as that is `None` of all of the bogus ports that I have seen.

so something like `for p in sorted([p for p in serial.tools.list_ports.comports() if p.location]):`